backend/clubs/migrations/0014_event_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.2 on 2020-01-15 00:02
```

Comment:
After you're ready to merge, merge all of these migrations into one! You can do so by deleting all of them and running `./manage.py makemigrations` again.